### DISCUSSION TITLE
Offline Update Issues

### DISCUSSION THREAD

**Discussion by ow111** [Security-Onion-Solutions/securityonion#2704] [Unsupported Versions] [answered]
Hi,

Trying to update SO via ISO in an offline environment, ever since running so-features-enable to allow us to generate reports this update process seems to fail as the services throw lots of errors when starting up. This is on the manager node is a distributed deployment. Is there a way to also pull the updates for the packages that were changed from enabling the so-features ?

Also ever since running the so-features-enable command we are no longer seeing logs in our kibana dashboard whereas we were before, Would that be to do with data being sent/received via cleartext? We are unable to change elasticsearch back to http transmission therefore seems that the manager node cannot connect to the search nodes.
`[2021-01-20T11:49:39,424][INFO ][logstash.outputs.elasticsearch] Elasticsearch pool URLs updated {:changes=>{:removed=>[], :added=>[https://x.x.x.x:9200/]}}
[2021-01-20T11:49:39,737][ERROR][logstash.javapipeline ] Pipeline error {:pipeline_id=>"search", :exception=>#<Manticore::UnknownException: Unsupported or unrecognized`

**Comment by TOoSmOotH** [comment, marked as answer]
Features + Airgap upgrade do not work currently as we don't distribute features on the ISO. There are some changes coming in 2.3.30 that will address this. In the meantime you will need internet access to get those containers.

**Reply by ow111** [reply]
Thanks!
